mis/shared/types.py

```diff
@@ -24,6 +25,32 @@ class MISInstance:
     def __init__(self, graph: networkx.Graph):
         # FIXME: Make it work with pytorch geometric
         self.graph = graph
+    
+    def draw(self, nodes: list[int] | None = None) -> None:
+        # Obtain a view of all nodes
+        all_nodes = self.graph.nodes
+        # Compute graph layout
+        node_positions = networkx.kamada_kawai_layout(self.graph)
+        # Keyword dictionaries to customize appearance
+        highlighted_node_kwds = {"node_color": "red", "node_size": 600}
+        unhighlighted_node_kwds = {"node_color": "white", "edgecolors": "black", "node_size": 600}
+        if nodes: # If nodes is not empty
+            if not set(nodes).issubset(all_nodes):
+                raise Exception("nodes must be a subset of nodes present in the problem instance")
```

Comment:
Could you clarify the error message so that it contains the index of "bad" nodes?

Reply:
Good idea. You could limit it to the first 10 entries, for instance.